spec/models/profile_spec.rb

```diff
@@ -323,59 +323,95 @@
     it 'activates a profile after password reset' do
       profile = create(
         :profile,
+        :password_reset,
         user: user,
-        active: false,
-        deactivation_reason: :password_reset,
       )
 
+      expect(profile.activated_at).to eq nil # to change
+      expect(profile.active).to eq false # to change
+      expect(profile.deactivation_reason).to eq 'password_reset' # to change
+      expect(profile.fraud_review_pending?).to eq(false)
+      expect(profile.gpo_verification_pending_at.present?).to eq false
+      expect(profile.initiating_service_provider).to eq nil
+      expect(profile.verified_at).to eq nil
+
       profile.activate_after_password_reset
 
-      expect(profile.active).to eq true
-      expect(profile.deactivation_reason).to eq nil
+      expect(profile.activated_at).to be_present # changed
```

Comment:
For this and other expectations where you're expecting a value to change, using the `change` matcher might make them clearer.  https://rubydoc.info/gems/rspec-expectationshttps://rubydoc.info/gems/rspec-expectations/RSpec%2FMatchers:change/RSpec%2FMatchers:change
```suggestion
expect(profile.activate_after_password_reset).to change{profile.activated_at}.from(nil)
```

Reply:
This is a good suggestion! I think I will leave this for now until we commit to which ones we expect to change and how.